app/i18n/en/sub.php

```diff
@@ -68,7 +68,7 @@
 		'error' => 'This feed has encountered a problem. Please verify that it is always reachable then update it.',
 		'export-as-opml' => array(
 			'download' => 'Download',
-			'help' => 'XML file',
+			'help' => 'XML file (restricted data. <a href="https://freshrss.github.io/FreshRSS/en/developers/OPML.html" target="_blank">See documentation</a>)',
```

Comment:
Is it a good wording?
![grafik](https://github.com/FreshRSS/FreshRSS/assets/1645099/54e83ad7-65c3-4d59-afdc-e8b96312e33d)

Reply:
I am not sure to understand what you mean by *restricted data*

Reply:
I want to highlight that not all data from above will be exported

Reply:
Ah, then maybe *subset* or *partial*  or *limited* or something like that